source/array-slice.d.ts

```diff
@@ -0,0 +1,62 @@
+import type {IsNegative} from './numeric';
+import type {Add, Lt, Lte, Max} from './internal';
+
+/**
+Returns a copy of a section of an array, just like `Array.slice` in js.
+
+@example
+```
+import type {ArraySlice} from 'type-fest';
+
+type T0 = ArraySlice<[0, 1, 2, 3, 4]>; 					//=> [0, 1, 2, 3, 4]
```

Comment:
```suggestion
type T0 = ArraySlice<[0, 1, 2, 3, 4]>;
//=> [0, 1, 2, 3, 4]

```

Same with the others.